.travis.yml

```diff
@@ -79,10 +85,15 @@ before_install:
       phpenv config-rm xdebug.ini || true;
     fi
   - if [[ "$COVERALLS_ENABLED" == "true" ]]; then
-      travis_retry composer require --dev satooshi/php-coveralls:^2.0 --no-update $COMPOSER_FLAGS;
+      travis_retry composer require --dev php-coveralls/php-coveralls:^2.0 --no-update $COMPOSER_FLAGS;
+    fi
+  - if [[ "$IMAGICK_ENABLED" == "true" ]]; then
+      pecl channel-update pecl.php.net;
+      yes | pecl install imagick;
+    fi
+  - if [[ "$ENQUEUE_ENABLED" == "false" ]]; then
+      composer remove --dev enqueue/enqueue-bundle;
```

Comment:
You should add the `--no-update` option as the `update` is performed at `install` step.